services/platform/app/components/icons/onedrive-icon.tsx

```
@@ -1,6 +1,8 @@
+import { cn } from '@/lib/utils/cn';
+
 export const OneDriveIcon = ({ className }: { className?: string }) => {
   return (
-    <svg viewBox="0 0 31 20" className={className} aria-hidden>
+    <svg viewBox="0 0 31 20" className={cn('size-full', className)} aria-hidden>
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)